introduction.md

```diff
@@ -1,3 +1,6 @@
 Shubhra Prakash
 Favorite Code School path is Javascript
-
+*Javascript 
+*HTML5
+*CSS3
+*Git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?